src/components/ui/members/member-container.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, CardContent } from "@/components/ui/card";
```

Comment:
These seem to be unused in this file.

```suggestion
```

Reply:
Good catch 1c1c86bfd8ab33f7f45819a79a06caaaf77cf48e